caffe2/CMakeLists.txt

```diff
@@ -487,6 +489,21 @@ if (NOT INTERN_BUILD_MOBILE OR NOT BUILD_CAFFE2_MOBILE)
     ${TORCH_SRC_DIR}/csrc/jit/mobile/type_parser.cpp
     )
 
+  if (USE_LLVM)
+    message(STATUS "Looking for LLVM in ${USE_LLVM}")
+    find_package(LLVM QUIET PATHS ${USE_LLVM} NO_DEFAULT_PATH)
```

Comment:
Most of the `find_package` live in `cmake/Dependencies.cmake`. I don't *really* care where you put it, but it might make it harder for people to find this logic in the future.